cmd/glasskube/cmd/install.go

```diff
@@ -279,6 +280,8 @@ func init() {
 	installCmd.PersistentFlags().StringVar(&installCmdOptions.Repository, "repository", installCmdOptions.Repository,
 		"specify the name of the package repository to install this package from")
 	installCmd.PersistentFlags().BoolVar(&installCmdOptions.NoWait, "no-wait", false, "perform non-blocking install")
+	installCmd.PersistentFlags().BoolVar(&installCmdOptions.DryRun, "dry-run", false,
```

Comment:
You can also mark `dry-run` and `no-wait` as mutually exclusive, since `dry-run` implies `no-wait`.